modules/mirror_environment/manifests/nrpe.pp

```diff
@@ -3,10 +3,16 @@
 # This class defines NRPE commands to be used with the pdxcat/nrpe module
 # this repository uses.
 
-class mirror_environment::nrpe{
+class mirror_environment::nrpe {
+
+  nrpe::command {'check_disk':
+    ensure  => present,
+    command => 'check_disk -w 10% -c 5%',
+  }
+
+  nrpe::command {'check_disk_mirror-data':
+    ensure  => present,
+    command => 'check_disk -w 10% -c 5% /srv/mirror_data',
```

Comment:
Should this use mirror_environment::mounts::mirror_data so that if the mount point moves the check continues to do the correct thing?